src/Command/InstallWebDirCommand.php

```diff
@@ -73,6 +73,7 @@ protected function executeLocked(InputInterface $input, OutputInterface $output)
         $webDir = rtrim($baseDir, '/').'/web';
 
         $this->addFiles($webDir);
+        $this->removeInstallPhp($webDir);
```

Comment:
To remove the file after an update from Contao 4.3. Otherwise the user will get an exception because the kernel does no longer work.